src/atcore.cpp

```diff
@@ -324,3 +324,35 @@ void AtCore::detectFirmware()
     connect(serial(), &SerialLayer::receivedCommand, this, &AtCore::findFirmware);
     requestFirmware();
 }
+void AtCore::statusChanged(const PrinterStatus &newStatus)
+{
+    emit(printerStatusChanged(newStatus));
+}
+/*~~~~~Control Slots ~~~~~~~~*/
+void AtCore::home()
+{
+    pushCommand(GCode::toCommand(GCode::G28));
+}
+
+void AtCore::home(QString axis)
+{
+    axis.replace(QChar::fromLatin1('X'), QStringLiteral("X0"));
+    axis.replace(QChar::fromLatin1('Y'), QStringLiteral("Y0"));
+    axis.replace(QChar::fromLatin1('Z'), QStringLiteral("Z0"));
+    pushCommand(GCode::toCommand((GCode::G28), axis));
+}
+
+void AtCore::setExtruderTemp(double temp, int extruder)
+{
+    pushCommand(GCode::toCommand(GCode::M104, QString::number(extruder), QString::number(temp)));
```

Comment:
Also, don't need to convert to number, since last PR from Patrick use QSring directly to form the QStringLiteral.